### PR TITLE
docs: add docs for running renku sessions on a remote machine

### PR DESCRIPTION
**Pull request by Panaetius**
closes https://github.com/SwissDataScienceCenter/renku-python/issues/3229

### CODE REVIEW

File: docs/how-to-guides/own_machine/session-running-remotely.rst [hunk 1]
```diff
@@ -0,0 +1,131 @@
...
+   f1693c198e  running   http://0.0.0.0:56674/?token=910ca732ef574049a22d41d0f1109f56
+
+.. note::
+
+   Note: Renku may inform you that the container image for the current commit
+   does not yet exist on your machine, and Renku will offer to build it for you. Say yes!
+
+.. note::
+
+    **Building too many images?**
```
Comment (rokroskar) [inline docs/how-to-guides/own_machine/session-running-remotely.rst:70]: This seems to belong in a more generic section about sessions

**Review by rokroskar — COMMENTED** [review]
Thanks @Panaetius, looks good - just some minor suggestions

File: docs/how-to-guides/own_machine/session-running-remotely.rst [hunk 2]
```diff
@@ -0,0 +1,131 @@
...
+
+To run a Renku session, you need to have `Docker installed <https://docs.docker.com/get-docker/>`_.
+
+On Linux systems, Docker requires root privileges. If you do not have root access to the host, you might
+consider `rootless Docker <https://docs.docker.com/engine/security/rootless/>`_. If the VM does not
+support Docker, you might need to ask an administrator to install it for you.
+
+If the remote machine you want to use is itself a Docker container, you'll need to enable
+``Docker in Docker``, either using the ``docker:dind`` image in ``priviledged`` mode or
+by mounting the host docker socket into your container.
```
[the remark this answers is not in the document]
Comment (Panaetius) [reply docs/how-to-guides/own_machine/session-running-remotely.rst:42]: I didn't find any official documentation, just blog posts, and I'm a bit hesitant of linking to some random blog post

**Comment by rokroskar** [comment]
I didn't know about this: https://docs.docker.com/engine/context/working-with-contexts/

Seems like that should be a much nicer way of using docker on another machine.

File: docs/how-to-guides/own_machine/session-running-remotely.rst (hunk 2 above)
[the remark this answers is not in the document]
Comment (Panaetius) [reply docs/how-to-guides/own_machine/session-running-remotely.rst:42]: Well dind also changed a bit (the DIND github repo is archived) as it is now officially supported via the `docker:dind` image from what I saw. Not sure if that influences the implications of using it, though.

[the remark this answers is not in the document]
Comment (rokroskar) [reply docs/how-to-guides/own_machine/session-running-remotely.rst:42]: btw this is what the docker hub page of the official docker image points to: https://docs.docker.com/engine/reference/run/#runtime-privilege-and-linux-capabilities

**Pull request merged**